k8s/names.go

```diff
@@ -10,9 +10,6 @@ import (
 // ToSubdomain converts a domain to a subdomain.
 // If the input is not a valid domain an error is thrown.
```

Comment:
It appears this is no longer a requirement? If not the comment could use an update.